cheatsheets/Secure_Product_Design_Cheat_sheet.md

```diff
@@ -0,0 +1,92 @@
+
+### 2. The principle of Defence in Depth
+
+The principle of Defence in Depth is a security strategy that involves multiple layers of security controls to protect an organization’s assets. It is based on the idea that if one layer of security fails, the other layers will still be able to protect the asset. The layers of security can include physical security, network security, application security, and data security. The goal of Defence in Depth is to create a secure environment that is resilient to attack and can quickly detect and respond to any security incidents. By implementing multiple layers of security, organizations can reduce the risk of a successful attack and minimize the damage caused by any successful attack.
+
+### 3. The principal of Zero Trust
+
+Zero Trust is a security model that assumes that all users, devices, and networks are untrusted and must be verified before access is granted. It is based on the idea that organizations should not trust any user, device, or network, even if they are inside the organization’s network. Instead, all requests for access must be authenticated and authorized before access is granted. Zero Trust also requires organizations to continuously monitor and audit user activity to ensure that access is only granted to those who need it. This model is designed to reduce the risk of data breaches and other security incidents by ensuring that only authorized users have access to sensitive data.
+
+### 4. The principal of Security in the Open
+
+Security in the Open is a concept that emphasizes the importance of security in open source software development. It focuses on the need for developers to be aware of the security implications of their code and to take steps to ensure that their code is secure. This includes using secure coding practices, testing for vulnerabilities, and using secure development tools. Security in the Open also encourages developers to collaborate with security experts to ensure that their code is secure. Finally, it emphasizes the need for developers to be aware of the security implications of their code and to take steps to ensure that their code is secure.
+
+## Security Focus Areas
+
+### 1. Context
+
+Where does this application fit into the ecosystem of the organisation, which departments use it and for what reason? What kinds of data might it contain, and what is the risk profile as a result?
+
+The processes employed to build the security context for an application include [Threat Modelling](Threat_Modeling_Cheat_Sheet.md) - which results in security related stories being added during **_Product Design_** at every iteration of *product delivery* - and when performing a Business Impact Assessment - which results in setting the correct Product Security Levels for a given product during **_Product Inception_**.
```

Comment:
Same spelling question here - in the US it's Modeling with one "l"

Reply:
It's spelled both ways. There's a lot of UK vs US spelling differences like that. We should at least be consistent for the same word. Ideally, we probably should be consistent for a given CS.

Reply:
Opted for US again here